flink-connectors/flink-connector-elasticsearch2/pom.xml

```diff
@@ -212,7 +212,7 @@ under the License.
 									<excludes>
 										<!-- Only some of these licenses actually apply to the JAR and have been manually
 										 	placed in this module's resources directory. -->
-										<exclude>META-INF/license</exclude>
+										<exclude>META-INF/license/**</exclude>
```

Comment:
I find that even we remove this exclude, the license folder of netty will not be packaged in the jar. Still, I can't figure out why. 

I also create a simple test project to verify this. In my test project, the license folder of netty will be packaged into the jar if we remote the exclude.

Reply:
On case-insensitive filesystems the license directory clashes with the LICENSE file and is excluded in any case.

Reply:
> On case-insensitive filesystems the license directory clashes with the LICENSE file and is excluded in any case.

Thanks a lot for the explanation. +1 for the PR.